libs/play/index.js

```diff
@@ -339,29 +339,49 @@ function playSubdomain(hostname) {
 export async function handleRunner(req, res) {
   const url = new URL(req.url, "https://example.com");
   if (url.searchParams.has("blank")) {
-    return res.setHeader("Content-Type", "text/html").status(200).end();
+    res.setHeader("Content-Type", "text/html").status(204).end();
+    return;
```

Comment:
I wouldn't make this change, I'm concerned it might not work as expected and go back to breaking things in Safari:

> The HTTP 204 No Content [successful response](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status#successful_responses) status code indicates that a request has succeeded, but the client **doesn't need to navigate away from its current page.**

Reply:
Reverted in a26f3cfbf770406278602ddccf5b94d6162973ed with a comment.